apiserver/facades/schema.json

```diff
@@ -22319,7 +22319,10 @@
         "Description": "ExternalControllerUpdaterAPI provides access to the CrossModelRelations API facade.",
         "Version": 1,
         "AvailableTo": [
-            "controller-machine-agent"
+            "controller-machine-agent",
+            "machine-agent",
+            "unit-agent",
+            "model-user"
```

Comment:
I wonder why these changed?

Reply:
So when I originally wrote this, all authorizers were the first action. It's clear that isn't the case any longer. We should fix this.

Reply:
done, I moved the check to the register and the schema changes are gone, thanks!